src/mono/netcore/System.Private.CoreLib/src/System/Reflection/Emit/EnumBuilder.Mono.cs

```diff
@@ -40,7 +40,9 @@ namespace System.Reflection.Emit
 {
     public sealed partial class EnumBuilder : TypeInfo
     {
+        [DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.All)]
```

Comment:
I'm curious - does the linker produce a warning when assigning the `new TypeBuilder` ? (I think it should...)

Reply:
Yes, there is a warning. I'll get to it in some future pass.

Right now I basically do this:
* Annotate dozen new places
* Get 9 new warnings
* Ignore them and create a pull request

CoreLib still has 128 warnings at this point even after all the pull requests I did. We started with ~140 warnings.

Reply:
Thanks - I read "we're making progress" - 😉  👍

Reply:
> Thanks - I read "we're making progress" - 😉 👍

Yes, I'll be bald soon.